File: xFit/xFit.Services/Database/OmiljeniProizvod.cs
using System;
using System.Collections.Generic;

namespace xFit.Services.Database;

public partial class OmiljeniProizvod
{
    public int OmiljeniProizvodId { get; set; }

    public DateTime? DatumDodavanja { get; set; }

    public int? ProizvodId { get; set; }

    public int? KlijentId { get; set; }

    public virtual Klijent? Klijent { get; set; }

    public virtual Proizvod? Proizvod { get; set; }
}

File: xFit/UI/xfit_mobile/lib/models/omiljeni_proizvod.dart
 import 'package:json_annotation/json_annotation.dart';

part 'omiljeni_proizvod.g.dart';

@JsonSerializable()
class OmiljeniProizvod{
	int? omiljeniProizvodId;
  DateTime? datumDodavanja;
	int? proizvodId;
	int? klijentId;

  OmiljeniProizvod(this.omiljeniProizvodId, this.datumDodavanja, this.proizvodId,this.klijentId);

  factory OmiljeniProizvod.fromJson(Map<String, dynamic> json) => _$OmiljeniProizvodFromJson(json);

  Map<String, dynamic> toJson() => _$OmiljeniProizvodToJson(this);

}